index.html

```diff
@@ -140,6 +140,7 @@ <h3>Order anything</h3>
 
     <section class="section-steps">
         <div class="row">
+            <!--Title is not formatted like the other titles-->
```

Comment:
I believe you need to add either margin top to the title, or add padding to the section-steps